testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -97,6 +104,160 @@ QUnit.testInActiveWindow('Form\'s inputs saves value on refresh', function(asser
     assert.deepEqual(formData, { name: 'test' }, 'value updates');
 });
 
+
+['xs', 'sm', 'md', 'lg'].forEach(screenSize => {
+    function createForm(testConfig) {
+        const config = extend({
+            colCountByScreen: { xs: 1, sm: 2, md: 3, lg: 4 },
+            items: [
+                { dataField: 'field1' }, { dataField: 'field2' }, { dataField: 'field3' }, { dataField: 'field4' }
+            ],
+        }, testConfig);
+        return $('#form').dxForm(config).dxForm('instance');
+    }
+
+    function getExpectedCallCount(size) {
+        const renderCount = { xs: 21, sm: 23, md: 25, lg: 22 };
```

Comment:
20+ call of a method :(
Do we really want to check the exact number of calls?

Reply:
Yep, it is redundant. Simplified tests and check only the called property of the stub